lntest/itest/lnd_single_hop_invoice_test.go

```diff
@@ -0,0 +1,177 @@
+// +build rpctest
```

Comment:
That is the scheme that was started. It comes from `lnd_test.go`, but perhaps it isn't so great. If we want to rename it, I would tack on a commit right before merge. To prevent rebases over moves. But as this PR is open for a while already and it isn't really in scope, I'd rather leave it for another time.

Reply:
sgtm